Car.cpp

```diff
@@ -0,0 +1,17 @@
+#include "Car.hpp"
+#include <iostream>
+
+void Car::turnLeft() {
+}
+void Car::turnRight() {
+}
+void Car::brake() {
+    currentSpeed_ = 0;
+}
+void Car::accelerate(int speed) {
+    currentSpeed_ += speed > 0 ? speed : 0;
```

Comment:
So you let's assume you have 120 speed. Then you can accelerate -90,because speed would be 30. Isn't brake supposed to slow down brake and accelerate only work for positive values?

Reply:
you don't accelerate negative values. It does not work for negative values. If a negative value was passed to it it just adds nothing to the current speed